docs/examples.md

```diff
@@ -109,21 +96,35 @@ _e("key"
 _e "key"
 ```
 
-Note1: we don't match the closing parenthesis as you may want to pass arguments to your translation function.
-
-Note2: the parser is smart about escaped single or double quotes you may have in your key.
+Please note that:
+- We don't match the closing parenthesis, as you might want to pass arguments to your translation function;
+- The parser is smart about escaped quotes (single or double) you may have in your key.
 
+### Command line
 
+```
+$ i18next /path/to/file/or/dir -f __,_e
+```
 
-**Change the regex (cli and gulp)**
-
-Command line:
+You can also pass this option using the config file (`i18next-parser.config.js`). Just add the `function` property to the related lexer:
+```js
+{
+  lexers: {
+    js: [{
+      lexer: 'JavascriptLexer',
+      functions: ['t', 'TAPi18n.__', '__']
+    }]
+  }  
+}
+```
 
-`i18next /path/to/file/or/dir -p "(.*)"`
+### Gulp
 
-Gulp:
+```js
+.pipe(i18next({functions: ['__', '_e']}))`
+```
 
-`.pipe(i18next({parser: '(.*)'}))`
+## Changing the regex
```

Comment:
This is also an artefact of a very old version. If someone want a "custom regex", they must write a custom lexer.

Reply:
I removed this section.